### ISSUE TITLE
[v1.1] Code refine for performance test

### ISSUE THREAD

**Issue by XinyaoWa** [intel/e2eAIOK#202]
- [ ] Model Adapter:
    - [ ] Add/update finetuner and distiller realted configuration files to meet performance test requirements
    - [ ] In task.py, do not create distiller/adapter/finetuner to match strategy, which can save useless definition
    - [ ] In train.py, disable record parameters in tensorboard, which can save a lot of time
    - [ ] Tensorboard dir will be created in default even user don't need it, fix it in main.py
- [ ] Common Lib
    - [ ] In CV dataloader, find evaluation data will be split in distribution mode, which is not proper in some cases. Add an option "ddp_eval_nosplit" to make a complete evaluation.
    - [ ]  For Earlystop in utils.py, disable tolerance when set target metric
